ko/admin/config.rst

```diff
@@ -2134,6 +2138,10 @@ HA 관련 파라미터
     
  **recovery_progress_logging_interval** 는 복구 (recovery) 의 상세과정을 로그에 출력할지 여부와 출력 주기를 설정한다. 분석(analysis), 리두(redo), 언두(undo) 각 단계별 전체 작업량과 현재 작업량, 앞으로 남은 시간 예상치를 출력한다. 5초보다 작게 설정될 경우 5초로 설정된다.
 
+**supplemental_log**
+
+ **supplemental_log** 는 CDC (Change Data Capture) 또는 :ref:`flashback` 지원에 필요한 정보를 로그 볼륨에 기록할지 여부를 결정한다. CDC와 flashback은 물리적 로그를 이용하여 트랜잭션이 데이터베이스를 논리적으로 어떻게 변경했는지 볼 수 있어야 한다. 이때 물리적 로그를 해석하는 데 필요한 추가 정보를 **supplemental_log** 를 이용하여 저장한다. 이 시스템 파라미터를 0보다 크게 설정하면 기존 트랜잭션 로그 외에도 더 많은 로그가 생성되고 저장되므로 성능에 영향을 끼친다. 이 시스템 파라미터를 1로 설정하면 사용자가 실행한 DML 및 DDL을 해석하는 데 필요한 정보가 기록된다. 또한 이 시스템 파라미터를 2로 설정하면 DML 해석에 필요한 정보만 기록된다.
```

Comment:
Just an opinion.
```suggestion
 **supplemental_log** 는 CDC (Change Data Capture) 또는 :ref:`flashback` 지원에 필요한 정보를 로그 볼륨에 기록할지 여부를 결정한다. CDC와 flashback은 물리적 로그를 해석하여 트랜잭션이 데이터베이스를 논리적으로 어떻게 변경했는지 추론한다. 이때 물리적 로그를 해석하기 위해서는 추가 정보가 필요하고 이를 위해 **supplemental_log** 를 사용한다. 이 시스템 파라미터를 0보다 크게 설정하면 기존 트랜잭션 로그 외에도 더 많은 로그가 생성되고 저장되므로 성능과 로그 저장공간에 영향을 끼친다. 이 시스템 파라미터를 1로 설정하면 사용자가 실행한 DML 및 DDL을 해석하는 데 필요한 정보가 기록된다. 또한 이 시스템 파라미터를 2로 설정하면 DML 해석에 필요한 정보만 기록된다.
```

Reply:
Thanks, I think it's clearer than before